website/docs/cloud-docs/projects/manage.mdx

```diff
@@ -0,0 +1,120 @@
+---
+page_title: Manage projects in HCP Terraform and Terraform Enterprise
+description: |-
+  Use projects to organize and group workspaces and create ownership boundaries
+  across your infrastructure.
+---
+
+# Manage projects
+
+This topic describes how to create and manage projects in HCP Terraform and Terraform Enterprise. A project is a folder containing one or more workspaces.
+
+## Requirements
+
+You must have the following permissions to manage projects:
+
+- You must be a member of a team with the **Manage all Projects**  permissions enabled to create a project. Refer to [Organization Permissions](/terraform/cloud-docs/users-teams-organizations/permissions#organization-permissions) for additional information.
+- You must be a member of a team with the update team access to add a new team access for organization-visible teams for a project.
+- You must be a member of a team with update and delete permissions to be able to update and delete teams respectively.
+
+To delete tags on a project, you must be member of a team with the **Admin** permission group enabled for the project.
```

Comment:
Again, bold here doesn't feel right since this isn't a UI element on this page? Also how would you feel about a link since the permission is kind of hidden?
```suggestion
To delete tags on a project, you must be member of a team with ["Project Admin" permissions](/terraform/cloud-docs/users-teams-organizations/permissions#project-admin) for that project.
```

Reply:
Also, to match formatting should we either make these list items or remove the bullet points above?